pg_ltree.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.rdoc']
   s.test_files = Dir['test/**/*']
 
-  s.add_dependency 'activerecord', '>= 4.0.0', '<= 7.0.0.rc1'
+  s.add_dependency 'activerecord', '>= 5.2', '<= 7.0'
```

Comment:
Shouldn't this be `< 8`, otherwise you can't do, say, Rails 7.0.1, only 7.0.0 exactly.

Reply:
I wasn't sure if you would want to allow support for 7.1 without testing first? But this probably wouldn't allow 7.0.1, so should change regardless. I'll change it to `< 8`